package org.bukkit.command;

public interface CommandSender {
    void sendMessage(String message);
    String getName();
    boolean hasPermission(String permission);
}
